src/common/contexts/themeContext/ThemeProvider.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from 'react';
+import { ThemeProvider } from 'styled-components';
+import { lightTheme, darkTheme } from './themes';
+import GlobalStyle from './global';
+
+export enum Theme {
+  Light = 'light',
+  Dark = 'dark',
+}
+type ThemeContext = { themeMode?: any; toggleTheme: () => void };
+type ThemeProviderProps = {
+  children: React.ReactNode;
+};
+
+export const ThemeContext = React.createContext<ThemeContext>(
```

Comment:
Instead of exporting the context, we can create a custom hook

```tsx
export function useTheme() {
  return React.useContext(ThemeContext)
}
```

Reply:
this has been updated